TsRandomizer/Randomisation/OrbDamageManager.cs

```diff
@@ -0,0 +1,131 @@
+﻿using System;
+using System.Collections.Generic;
+using Timespinner.GameAbstractions.Inventory;
+using Timespinner.GameAbstractions.Saving;
+using Timespinner.GameObjects.Heroes;
+using TsRandomizer.Extensions;
+using TsRandomizer.IntermediateObjects;
+
+namespace TsRandomizer.Randomisation
+{
+	struct OrbDamageRange
```

Comment:
plz use tabs instead of spaces, i added an .editorconfig that should have prevented this